examples/jdbc-query-column-mapping/jdbc_query_column_mapping.md

```diff
@@ -0,0 +1,11 @@
+# Query with complex types
```

Comment:
```suggestion
# Query with a different column name mapping
```

Reply:
Fixed in [c54d904](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/c54d904a2003ddd6c534cadf8b0a0cf1f2572285)